hugo/content/docs/learn/workflow/_index.md

```diff
@@ -0,0 +1,51 @@
+---
+title: "Our workflow"
+weight: 0
+url: /docs/learn/worflow
+---
+
+## Workflow diagram
+
+{{<mermaid>}}
+flowchart TD
+  A(["1. Install Yeoman generator"]);
+  B(["2. Scaffold a Langium project"]);
+  C(["3. Write the grammar"]);
+  D(["4. Generate AST files"]);
+  E(["5. Resolve cross-references"]);
+  F(["6. Create validations"]);
+  G(["7. Generate what you want"]);
```

Comment:
Each sub step should have a way of validation as far as possible. Like a hint of how to write a test for it.